src/main/java/com/jvm_bloggers/frontend/public_area/rss/RssInformationPage.java

```diff
@@ -0,0 +1,72 @@
+package com.jvm_bloggers.frontend.public_area.rss;
+
+import com.jvm_bloggers.frontend.public_area.AbstractFrontendPage;
+import com.jvm_bloggers.frontend.public_area.HomePage;
+
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.apache.wicket.request.Url;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.wicketstuff.annotation.mount.MountPath;
+
+import java.util.Arrays;
+
+import static com.jvm_bloggers.core.rss.BlogPostsController.RSS_FEED_MAPPING;
+
+@MountPath("rss")
+public class RssInformationPage extends AbstractFrontendPage {
+    private static final String FEED_URL_PATTERN = "%s%s";
+
+    private final String feedUrl;
+
+    public RssInformationPage() {
+        this.feedUrl = String.format(FEED_URL_PATTERN, getRootUrl(), RSS_FEED_MAPPING);
+
+        add(new ExternalLink("rss_feed_url", feedUrl, feedUrl));
+
+        Arrays.stream(FeedParameter.values())
+            .forEach(e -> add(getExternalLink(e)));
```

Comment:
It's not a blocker. I only wanted to show a different approach to `Stream` processing which, by 
 * splitting processing into atomic steps
 * eliminating unnecessary variables (by replacing direct invocations with method references)

Makes the code much more readable and self-explanatory almost like natural language:
```java
 Arrays.stream(FeedParameter.values())
    .map(this::getExternalLink)
    .forEach(this:add);
```
WDYT?